plugins/main/windows/l2bridge/l2bridge_windows.go

```diff
@@ -100,14 +100,22 @@ func cmdAdd(args *skel.CmdArgs) error {
 
 		// NAT based on the the configured cluster network
 		if n.IPMasq {
-			n.ApplyOutboundNatPolicy(n.clusterNetworkPrefix.String())
+			n.ApplyOutboundNatPolicy(n.ClusterNetworkPrefix)
+			cmd := exec.Command("powershell", 
```

Comment:
And this is how creating a new NetNat can be worked around and making NAT fully functional with kubernetes on windows

Reply:
@ptylenda 

Hi, I manually build host-local.exe and l2bridge from this pr,  these binaries will auto create NetNat rule for me. The pod on windows can reach outside address but the pod-to pod communication is broken when these pods locate on  different windows node. After I manually delete these nat rule, the pod-to-pod communication is recovered but the pod can not reach outside addresses.